enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -88,10 +104,19 @@ func fileDigest(filePath string) (*repb.Digest, error) {
 
 // Snapshot holds a snapshot manifest along with the corresponding cache key.
 type Snapshot struct {
-	key      *fcpb.SnapshotKey
+	writeKey *fcpb.SnapshotKey
+	readKey  *fcpb.SnapshotKey
 	manifest *fcpb.SnapshotManifest
 }
 
+func (s *Snapshot) GetWriteKey() *fcpb.SnapshotKey {
```

Comment:
I don't see this used anywhere yet, that intentional?

Reply:
woops, reverted this readKey vs writeKey stuff. This key is only used in UnpackSnapshot so it's already readonly and it's the one associated w/ the manifest.